app/views/schools/partnerships/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: schools_cohort_path("2021"))
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-m">2021</span>
```

Comment:
We should be able to get the correct cohort from the URL, no need to hardcode I don't think